webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const SRC_DIR = path.join(__dirname, '/client/src');
+const DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  devtool: 'source-map',
+  module: {
+    rules: [
+      {
+        test: [/\.jsx$/],
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-react', '@babel/preset-env']
+          }
+        }
+      },
+    ]
+  },
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  }
+};
```

Comment:
You don't have anything here to load your webfont files, like your `.woff` files. Checkout my webpack config for an example on how to handle this!